subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/repositories/DefaultIvyArtifactRepository.java

```diff
@@ -243,6 +255,55 @@ public IvyArtifactRepositoryMetaDataProvider getResolve() {
         return metaDataProvider;
     }
 
+    @Override
+    public Map<RepositoryPropertyType, ?> getProperties() {
```

Comment:
:x: Let's split this guy into different methods for each property, for readability.

Reply:
done